File: Logica de Programação/Contador.java
import java.util.Scanner;

public class Contador {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        int num, maior, cont;

        System.out.println("Digite 5 números: ");

        // Lê o primeiro número e inicializa o maior com ele
        num = entrada.nextInt();
        maior = num;

        // Laço para ler os próximos 4 números
        for(cont = 1; cont <= 4; cont++ ){
            num = entrada.nextInt();

            // Verifica se o número atual é maior que o maior armazenado
            if (num > maior) {
                maior = num;
            }
        }

        entrada.close(); // Fecha o Scanner após a leitura de todos os números

        // Exibe o maior valor digitado
        System.out.println("Maior valor = " + maior);
    }
}

File: Logica de Programação/Contador.py
# Solicita o primeiro número e inicializa o maior com ele
num = int(input("Digite 5 números: "))
maior = num

# Laço para ler os próximos 4 números
for cont in range(1, 5):
    num = int(input())

    # Atualiza o maior se o número atual for maior
    if num > maior:
        maior = num

# Exibe o maior valor digitado
print("Maior valor =", maior)
